module4/projects/capstone/evaluation.md

```diff
@@ -0,0 +1,185 @@
+  * Seeks clarification when requirements or tasks are unclear
+  * Demonstrates active listening skills during team discussions and meetings
+  * Shows willingness to help team members when they encounter difficulties
+- **Approaching Expectations:**
+  * Updates the project board about once per day
+  * Provides constructive and actionable feedback on at least 2 pull requests
+  * Ineffective time boxing: Spends too much time trying to figure things out on their own before asking for help
+- **Below Expectations:**
+  * Updates project board less than once per day
+  * Provides constructive and actionable feedback on 1 or fewer pull requests
+  * Behaving in an anti-collaborative way. This might look like:
+    * Not contributing to project planning or decision making discussions
+    * Disappearing for 24 hours or more (Monday-Friday) without notice or explanation
+    * Not following the project's defined workflow
+    * Missing deadlines, meetings, or discussions without notice more than once
+</section>
+
+<section class="dropdown">
+### Code Quality
+- **Exceeds Expectations:**
+  * Proactively seeks to understand and improve code performance by asking questions and researching solutions
+  * Shows a keen interest in learning about tools and architecture solutions outside of the scope of lessons taught at Turing
+  * Actively participates in reviews of their own code and addresses feedback effectively
+  * 100% of code is well tested, including sad paths and edge cases
+- **Meets Expectations:**
+  * Demonstrates strong understanding of the primary programming language and framework used in the project through implementation and successful completion of all issues assigned
+  * Effectively addresses feedback from code reviews
+  * At least 90% of code is well tested, including sad paths and edge cases
+  * Implements proper error handling
+  * Demonstrates understanding of code organization principles (e.g., separation of concerns, DRY)
+  * Maintains a clean and organized codebase, following agreed-upon coding standards
+- **Approaching Expectations:**
+  * Usually demonstrates a strong understanding of the primary programming language and framework used in the project though occassionally strays from conventions or the project's coding standards
+  * Usually addresses feedback from code reviews but occassionally misses details
+  * Improperly handles some errors
+  * Most code is well tested but some edge cases or sad paths are not tested
+- **Below Expectations:**
+  * Does not demonstrate a strong understanding of the primary programming language and framework used in the project either through incorrect implementation or lack of completion of issues assigned
+  * Does not follow the project's coding standards
+  * More often than not, does not implement error handling
+  * Code is not well tested missing many edge cases and sad paths
+</section>
+
+<section class="dropdown">
+### Technical Problem Solving
```

Comment:
I like the addition of this category! Great mechanism to highlight areas of growth with computational thinking, troubleshooting and research, etc.